### PR TITLE
[ZIPFLDR] Add some bitmap resources CORE-17092

### PR DESCRIPTION
**Pull request by oleg-dubinskiy**
## Purpose

Add some bitmap images to our zipfldr shell extenson. MS version of this has them, but our one currently hasn't. And I think without bitmaps the wizard dialogs look a bit incompleted.
I made them similar to MS bitmaps. They have exactly the same size (height x width), but looks differently, in ReactOS Tango style.

JIRA issue: [CORE-17092](https://jira.reactos.org/browse/CORE-17092)

## Proposed changes

- Add the resource bitmaps into res directory;
- Add accodring definitions into resource.h;
- Enable the bitmaps in zipfldr.rc using the definitions numbers;
- Enable the first bitmap (zipfldr1.bmp) in the IDD_PROPPAGEDESTINATION and IDD_PROPPAGECOMPLETE wizard dialogs for all localizations;
- Fix controls posision of those dialogs after adding the bitmaps same for all localizarions.

## TODO

- [x] Enable the 2nd bitmap (zipfldr2.bmp) at the right top of the wizard dialogs, same as it done in MS implemetation. (where I need to properly enable it if it should be outside those dialogs, probably near the caption instead of the BEGIN...END section?)
Also I haven't used the 3rd bitmap (zipfldr3.bmp) anywhere, since the dialog which uses it is not currently implemented in our zipfldr, compared to MS version. So I only added it as a resource file.

## Result

Before:
before_1
before_2

After:
after_1
after_2

Win2k3 zipfldr (for comparison):
MS_zipfldr_1
MS_zipfldr_2
MS_zipfldr_3

### CODE REVIEW

**Comment by feel-the-dz3n** [comment]
Shouldn't wizard's watermark be used here? https://docs.microsoft.com/en-us/windows/win32/controls/wizards

**Comment by oleg-dubinskiy** [comment]
Oh, looks like it should. Probably it's only way to properly enable zipfldr2.bmp in that wizard.

**Comment by gonzoMD** [comment]
@feel-the-dz3n
So far it is OK as is. The watermark would only be used on the (currently missing) welcome page of the wizard, the other pages use a static bmp (also on win2k3). 
But you are right regarding the banner image which will be defined in the pszbmHeader property of the PROPSHEETHEADER structure.

**Comment by oleg-dubinskiy** [comment]
Thanks to @gonzoMD's help, I finally enabled the small bmp in the headers of wizard pages and also improved the names of bmp resources. The current ones will be indeed better.
Now zipfldr wizard pages look as expected:
fixed-1
fixed-2

**Comment by gonzoMD** [comment]
Now only the Portuguese translation needs the layout changes. The rest is good. The missing welcome page can be done in another PR.

File: dll/shellext/zipfldr/zipfldr.rc [hunk 1]
```diff
@@ -3,6 +3,9 @@
 
 #include "resource.h"
 
+164 BITMAP "res/zipfldr1.bmp"
+200 BITMAP "res/zipfldr2.bmp"
+201 BITMAP "res/zipfldr3.bmp"
```
[the remark this answers is not in the document]
Comment (katahiromz) [reply dll/shellext/zipfldr/zipfldr.rc]: Fixed.

**Comment by JaredSmudde** [comment]
The English dialog is having issues. The extraction complete dialog is fine, it's just the first dialog.

english

**Pull request merged**